fullstop/src/main/resources/db/migration/V6__create_outdated_ami_type.sql

```diff
@@ -0,0 +1,20 @@
+INSERT INTO fullstop_data.violation_type (
```

Comment:
you can use the stored procedure defined in a prior migration script to create new violation types.